/*
 * Copyright (C) Mellanox Technologies Ltd. 2001-2013.  ALL RIGHTS RESERVED.
 *
 * This software product is a proprietary product of Mellanox Technologies Ltd.
 * (the "Company") and all right, title, and interest in and to the software product,
 * including all associated intellectual property rights, are and shall
 * remain exclusively with the Company.
 *
 * This software is made available under either the GPL v2 license or a commercial license.
 * If you wish to obtain a commercial license, please contact Mellanox at support@mellanox.com.
 */

/* disable pbuf_pool, it is not in use, and cosume a lot of memory. */

--- orig_lwip/arch/lwipopts.h	2013-03-08 13:59:04.113894000 +0200
+++ lwip/arch/lwipopts.h	2013-03-08 14:00:53.623631000 +0200
@@ -200,7 +200,7 @@
 #ifdef _LWIP_MIN_MEM_MODE
 #define PBUF_POOL_SIZE                  32
 #else
-#define PBUF_POOL_SIZE                 	131072 //256000 
+#define PBUF_POOL_SIZE                 	0 //256000
 #endif
 
 /**
